app/services/hearings_creator.rb

```diff
@@ -51,17 +55,41 @@ def push_appeals
                   appeal_data: {
                     appeal_type: appeal.dig(:type, :applicationCode),
                     appeal_outcome: appeal[:applicationOutcomes],
-                  })
+                  },
+                  application_data: nil,
+                  function_type: nil)
     end
   end
 
-  def push_to_sqs(shared_time:, case_urn:, defendant:, appeal_data:)
+  def push_applications
+    hearing[:courtApplications]&.each do |application|
+      defendant_cases = application.dig(:applicant, :masterDefendant, :defendantCase)
+      return if defendant_cases.nil?
+
+      defendant_cases.each do |defendant_case|
+        defendant = LaaReference.find_by(defendant_id: defendant_case[:defendantId], linked: true)
```

Comment:
Please make sure the variable name matches the thing that actually gets stored in it. When I see `defendant`, I expect a `Defendant` object of some kind to be in it.